src/test/java/edu/gatech/VRDR/MessagingTest.java

```diff
@@ -637,7 +639,20 @@ public void testCreateAckForDeathRecordVoidMessage() {
         assertEquals(voidMessage.getNCHSIdentifier(), ack.getNCHSIdentifier());
         assertEquals(voidMessage.getBlockCount(), ack.getBlockCount());
     }
-
+	
+    public void testSetBlockCountDeathRecordVoidMessage() {
+	DeathRecordVoidMessage voidMessage = new DeathRecordVoidMessage();
+	Integer value = null;
+	voidMessage.setBlockCount(value);
+	assertEquals(voidMessage.getBlockCount(), null);
+	value = 2;
+	voidMessage.setBlockCount(value);
+	assertEquals(voidMessage.getBlockCount(), 2); 
+    }
+
+//import org.hl7.fhir.r4.model.IntegerType;
+//import org.hl7.fhir.r4.model.UnsignedIntType;
+	
```

Comment:
Please delete these comments, also do a test with block count 0 or negative to make sure it doesn’t set the value